README.md

```diff
@@ -1,7 +1,20 @@
-# Data generation scripts
+# DBMS Benchmarking
+- 37,495,494 total rows imported into Cassandra (around 1 hr 15 min)
+- 37,500,494 total rows imported into PostgresSQL (not sure why this number is higher)
```

Comment:
Potentially have duplicate rows when generating large amounts of data in faker. higher likelyhood that this could happen with a large N